sources/vcdMaker/src/vcdMaker.cpp

```diff
@@ -57,7 +58,7 @@ int main(int argc, const char *argv[])
         }
```

Comment:
**Comment for line 55 above.** Again, `.empty()`. I think it would be nice to check all other places that use `.length()`.